aep/general/0211/aep.yaml

```
@@ -1,6 +1,7 @@
 ---
 id: 211
 state: approved
+slug: auth
```

```suggestion
slug: authz
```

...or "authorization"?  "auth" typically includes both authn/authentication and authz/authorization, right?  Or at least, that's how I've seen the term used.

---

definitely, good points. I'll fix the slug to be the full string "authorization", while allowing for the "authz" redirect.